internal/version/version.go

```diff
@@ -21,7 +21,7 @@ import (
 )
 
 // Version is the version of the build.
-const Version = "1.19.0-dev"
+const Version = "1.19.0-rc1"
```

Comment:
Do we wanna follow Kubernetes conventions and call it 
```suggestion
const Version = "1.19.0-rc.1"
```